.azure-pipelines.yml

```diff
@@ -269,6 +269,34 @@ jobs:
 #################################################################################
 
 
+- job: 'linux_pypi_build'
+  displayName: 'Linux PyPI build'
+  condition: startsWith(variables['build.sourceBranch'], 'refs/tags/')
+  timeoutInMinutes: 60
+  pool:
+    vmImage: 'ubuntu-18.04'
+  steps:
+  - checkout: self
+    submodules: true
+  - bash: |
+      set -eux
+      sudo apt install python python-pip
```

Comment:
`sudo apt install twine` to avoid the two steps ?

Reply:
Is there really an advantage to use twine installed through apt instead of pip?

Reply:
http://manpages.ubuntu.com/manpages/cosmic/man1/twine.1.html
Even the ubuntu documentation recommend to use pip :/

Reply:
amen ! ^^